### PR TITLE
Add java helper as for nodejs

### PR DESCRIPTION
**Pull request by gcollin**
Seems like some other PR removed it (my fault)

### CODE REVIEW

**Comment by yalh76** [comment]
Can you rename it to ynh_install_java to be more consistent to others language install

File: ynh_java/ynh_java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    #get the architecture: amd or arm ?
+    ynh_get_architecture
+
+    # Get the absolute path of this version of java
+    java_path="/usr/lib/jvm/java-$java_version-openjdk-$ynh_architecture/bin"
+
+    # Allow alias to be used into bash script
+    shopt -s expand_aliases
+
+    # Create an alias for the specific version of node and a variable as fallback
```
Comment (yalh76) [inline ynh_java/ynh_java]: there is some references of node instead of java ...

**Comment by yalh76** [comment]
ynh_remove_java is missing

**Comment by yalh76** [comment]
you should better base this helper from ynh_install_go of ynh_install_ruby

**Comment by gcollin** [comment]
Ok, I'll check with install_go and updates

**Comment by gcollin** [comment]
@yalh76 I renamed the scripts and added ynh_remove_java as well. However, I don't need to clean-up versions like for go because Java is installed using the yunohost package dependencies. It's removed when no other apps are using it.

**Review by yalh76 — APPROVED** [review]
Great work, thanks a lot :)

File: ynh_java/ynh_java (hunk 1 above)
Comment (yalh76) [reply ynh_java/ynh_java]: fixed

**Pull request merged**